tests/unit/test_charm.py

```diff
@@ -301,7 +301,7 @@ def test_relations_available_and_config_pushed_and_pebble_layer_already_applied_
             mounts={
                 "config_dir": Mount(
                     "/free5gc/config/nssfcfg.conf",
-                    Path(__file__).parent / "expected_config" / "config.conf",
+                    Path(__file__).parent / "actual_config" / "config.conf",
```

Comment:
After adding `.strip()` I don't think we need this file anymore. I checked on both my laptop and GH and it works fine.

Reply:
It won't fail, at least for now, but this test expects that config not to change, while it is written by other tests, this means that changing a test case might cause this test to fail without any change in the logic.